src/index.js

```diff
@@ -38,4 +39,8 @@ angular.module('vlui', [
       T: 'time',
       G: 'geo'
     }
+  })
+  .config(function (AnalyticsProvider) {
+    AnalyticsProvider
+      .setAccount({ tracker: 'UA-44428446-4', name: 'voyager', trackEvent: true });
```

Comment:
Given that `vega-lite-ui` is a library, not an application in and of itself, I'd expect there to be a configuration method for this rather than a config file itself—the configuration I'd assume to live in Voyager or Polestar, which would then pass it in to vlui as configuration.

This is the basic approach I've taken when I've extracted libraries out of larger applications, such as my mbtapi subway predictions client:
```js
var mbtapi = require('mbtapi').create({
  apiKey: 'yourAPIkeyGoesHere'
});
```
(`create` was relevant in my app, `config()` or `analyticsConfig()` may be more appropriate in this case)